examples/image_filters_using_image2d_t.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python3
+# this example shows how to blur, gray scale or brightness adjust the picture
+# using pyopencl and  image2d_t buffer objects
+# this python script will ask you for:
+# input image path
+# number of which filter you want to apply to
+# if gaussian blur is chosen it will ask you for parameters of gaussian kernel
+# if brightness adjustment is chosen it will ask you for a scaler that will
+# determines how much to adjust it
+# this script was my homework that was given on course in Parallel Algorithms
+# for more info contact me at bbozic13023rn@raf.rs
+__copyright__ = "Copyright (C) 2025 Bogdan Bozic"
+
+__license__ = """
+Permission is hereby granted, free of charge, to any person
+obtaining a copy of this software and associated documentation
+files (the "Software"), to deal in the Software without
+restriction, including without limitation the rights to use,
+copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following
+conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+"""
+import os
+import sys
+
+import numpy as np
+from PIL import Image
+
+import pyopencl as cl
+
+
+platform = cl.get_platforms()[0]
+device = platform.get_devices()[0]
+context = cl.Context([device])
+queue = cl.CommandQueue(context)
```

Comment:
```suggestion
context = cl.create_some_context()
queue = cl.CommandQueue(context)
device = queue.device
```
? Most examples just use `create_some_context`, which also takes into account environment variables and things.